docs/apps/crypto-trading.md

```diff
@@ -3,34 +3,46 @@ sidebar_position: 2
 title: Crypto Trading Bot
 ---
 
-### On GDN
+### Crypto App details
 
 | **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
 |----------- |----------|----|-----------|-----------|
 |  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/tutorial-cryptotrading/) |[github](https://github.com/Macrometacorp/tutorial-cryptotrading)|
 
-### On GDN (Wavelength 5G)
-
-| **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
-|----------- |----------|----|-----------|-----------|
-|  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/demo-crypto-trading-5g/) |
 
 :::note
 Stream Workers is currently an Enterprise only feature. 
 Please contact support@macrometa.com if you have any questions.
 :::
 
-**Stream App:**
+### Configuring the Crypto App
+
+1. If you haven't already, [signup](https://auth.paas.macrometa.io/signup) for a free Macrometa Playground account. 
+
+2. Login into the [Macrometa Console](https://auth.paas.macrometa.io/) Go to the `Collections` tab and create a new `Doc` collection named `trades` with `collection stream enabled` checked.
```

Comment:
```suggestion
2. Log in to the [Macrometa Console](https://auth.paas.macrometa.io/) Go to the Collections tab and create a new `Doc` collection named `trades` with `collection stream enabled` checked.
```

Reply:
This is actually three steps. Please break it up.

Also, "login" is a noun. "Log in" is a verb.

Reply:
Per the [Google Developer Style Guide](https://developers.google.com/style/ui-elements), UI items that you click are bold, not code style.